doc/Extensions/Graylog.md

```diff
@@ -17,7 +17,7 @@ standalone appliance.
 Config is simple, here's an example:
 
 ```php
-$config['graylog']['server']   = 'http://127.0.0.1';
+$config['graylog']['server']   = 'http://yourserver/search/universal/relative';
```

Comment:
This variable stays the same. It's `$config['graylog']['base_uri'] = '/somepath/';` that needs this setting.

We could default it to this new path if people specify the version of graylog they use. Or alternatively just say that they need to specify the base_uri.